src/main/java/com/google/sps/Utils.java

```diff
@@ -144,21 +173,21 @@ public int compare(Entity a, Entity b) {
         }
       };
 
-  /** creates a comparator based on entity dot product with a given vector. */
-  public static Comparator<Entity> getComparatorByInterestMatch(Map<String, Integer> metrics) {
-    Comparator<Entity> result =
-        new Comparator<Entity>() {
-          @Override
-          public int compare(Entity a, Entity b) {
-            if (!a.getKind().equals("Event") || !b.getKind().equals("Event")) {
-              throw new IllegalArgumentException("must be event items");
-            }
-            Map<String, Integer> v1 = Interactions.buildVectorForEvent(a);
-            Map<String, Integer> v2 = Interactions.buildVectorForEvent(b);
-            return Integer.compare(
-                Interactions.dotProduct(v1, metrics), Interactions.dotProduct(v2, metrics));
-          }
-        };
-    return result;
-  }
+  //   /** creates a comparator based on entity dot product with a given vector. (not used) */
```

Comment:
Please remove commented code in production. You can recover it from git history if you need it later.

Reply:
Got it!